test/test_vtx.py

```diff
@@ -146,7 +156,13 @@ def test_vtx_temperature(T, tmpdir):
     my_model.settings.stepsize = F.Stepsize(initial_value=1)
 
     my_model.initialise()
-    assert len(my_model._vtxfiles) == 1
+
+    vtx_exports = []
+    for export in my_model.exports:
+        if isinstance(export, F.ExportBaseClass):
+            vtx_exports.append(export)
+
+    assert len(vtx_exports) == 1
```

Comment:
Could you not simplify that by:

```python

all_vtx = [e for e in my_model.exports if isinstance(e, F.ExportBaseClass]
assert len(all_vtx) == 1
```

That comment applies for lines 120-126 too